test/Microsoft.Framework.Runtime.Tests/ProjectFilesCollectionFacts.cs

```diff
@@ -59,6 +59,30 @@ public void FilePatternsWorkForArraysAreSet()
             Assert.Equal(new[] { "a.cs", @"foo.js" }, target.ResourcesPatterns);
         }
 
+
+        [Fact]
+        public void FilePatternsDoNotIncludeComments()
+        {
+            var rawProject = Deserialize(@"
+{
+    ""code"": [""*.cs"", /* FOO */ ""../*.cs""],
+    ""exclude"": [""buggy/*.*"", /* FOO */],
```

Comment:
BTW there are a couple of these that aren't really valid - there's an `undefined` last element in the array because there's a trailing comma. There's another example below in `shared`.